_data/students.yml

```diff
@@ -400,3 +400,9 @@
   image: zahari.jpg
   facebook: zari.asparuhov
   bio: 17, Bulgaria, web developing and web servers with NodeJS
+
+- name: Srijan Jha
+  github: SrijanJha28
+  image: Srijan Jha.jpg
```

Comment:
Please don't use space in the URL. Please also edit  your image file name and remove the space,